examples/appgw/example.tfvars

```diff
@@ -1,4 +1,5 @@
-# --- GENERAL --- #
+### GENERAL ###
```

Comment:
I'm not convinced for `###`. 
Why we cannot use default comments for HCL (https://developer.hashicorp.com/terraform/language/syntax/configuration) e.g. single `#` ? 

Moreover in VScode we have short-cut for commenting code in HCL and this short-cut is also doing `#`

Reply:
I know single `#` is enough to mark a comment but I wanted to emphasise somehow that this is a **section block**. In `main.tf` files I also wanted to differentiate that from TF Registry reference comments.

Reply:
ok, it's clear. Thanks.